README_DRAFT.md

```diff
@@ -0,0 +1,95 @@
+# Open Platform for AI (OpenPAI) ![alt text][logo]
+
+[logo]: ./pailogo.jpg "OpenPAI"
+
+[![Build Status](https://travis-ci.org/Microsoft/pai.svg?branch=master)](https://travis-ci.org/Microsoft/pai)
+[![Coverage Status](https://coveralls.io/repos/github/Microsoft/pai/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/pai?branch=master)
+
+OpenPAI is an open source platform that provides complete AI model training and resource management capabilities, it is easy to extend and supports on-premise, cloud and hybrid environments in various scale. 
+
+# Table of Contents
+1. [When to consider OpenPAI](#when-to-consider-openpai)
+2. [Why choose OpenPAI](#why-choose-openpai)
+3. [How to deploy](#how-to-deploy)
+4. [How to use](#how-to-use)
+5. [Resources](#resources)
+6. [Get Involved](#get-involved)
+7. [Contributing](#contributing)
+
+## When to consider OpenPAI
+1. When your organization need a better utilization of AI computing resources (GPU/CPU etc.). On demand allocates resources. 
+2. When your organization need an easy IT ops platform for AI.
+3. When your organization need to share and reuse common AI assets like Model, Data, Environment, etc.
+4. When you want to run complete training pipeline in one place. 
+
+
+## Why choose OpenPAI
+The platform incorporates the mature design that has a proven track record in Microsoft's large-scale production environment.
+
+### Support on-premises and easy to deploy
+
+OpenPAI is a full stack solution. OpenPAI not only supports on-premises, hybrid, or public Cloud deployment, but also supports single-box deployment for trial users.
+
+### Support popular AI frameworks and heterogeneous hardware
+
+Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardware. Support Distributed training, such as distributed TensorFlow.
+
+### Most complete solution and easy to extend
+
+OpenPAI is a most complete solution for deep learning, support virtual cluster, compatible Hadoop / kubernetes eco-system, complete training pipeline at one cluster etc. OpenPAI is architected in a modular way: different module can be plugged in as appropriate. 
+
+
+## How to deploy
+#### 1 Prerequisites
+Before start, you need to meet the following requirements:
+
+- Ubuntu 16.04
+- Assign each server static IP address. Network reachable between servers.
```

Comment:
Assign each server **a** static IP address. Network **is** reachable between servers.